incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/config/OtlpEndpointAdapter.java

```diff
@@ -60,7 +61,7 @@ public OtlpEndpointConfig adaptFromJson(
     {
         String protocol = object.containsKey(PROTOCOL_NAME)
             ? object.getString(PROTOCOL_NAME)
-            : null;
+            : DEFAULT_PROTOCOL;
```

Comment:
Please make sure the default value is captured in the JSON schema patch as well.

Reply:
done